src/main/java/com/snowflake/kafka/connector/SnowflakeSinkConnectorConfig.java

```diff
@@ -302,14 +303,24 @@ static ConfigDef newConfigDef() {
             1,
             ConfigDef.Width.NONE,
             JVM_PROXY_PORT)
+        .define(
```

Comment:
is there any issue since we update the orderInGroup?

Reply:
I will need @sfc-gh-japatel to comment on that.

Reply:
If you're not sure, we can always add it at the end of this group?

Reply:
Sorry, I totally misunderstood what you were saying here. 
No, I'm pretty confident this isn't an issue. I initially looked at the Kafka docs to understand what was going on here in the code, and this isn't too important. It's just a way to order our parameters when they're displayed. I'm guessing things like Kafka Conluent Control Center will use this info to display a sink connector's properties in the UI in the same order that we defined them in. It's why I specifically inserted this new parameter in the middle. In my mind it made sense to have it come after proxy host and port, rather than at the end after proxy username and password.

Reply:
+1, no issues on changing the order, it is used in confluent UI